utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -0,0 +1,81 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import org.oppia.util.R
+import javax.inject.Inject
+import javax.inject.Singleton
+
+// TODO (#169) : Replace this with exploration asset downloader
+/** UrlImage Parser for android TextView to load Html Image tag. */
+@Singleton
+class UrlImageParser @Inject constructor(
+  @DefaultGcsPrefix private val gcsPrefix: String,
+  @DefaultGCSResource private val gcsResource: String,
+  @ImageDownloadUrlTemplate private var imageDownloadUrlTemplate: String
+) : Html.ImageGetter {
+
+  lateinit var htmlContentTextView: TextView
+  lateinit var context: Context
+  lateinit var entityType: String
```

Comment:
Can be made `private`

Reply:
+1 Please make all values private here since none need to be exposed outside of this class.

Reply:
done.